### PR TITLE
mypaint: 1.1.0 -> 1.2.1

### PR DESCRIPTION
**Pull request by jtojnar**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @goibhniu

### CODE REVIEW

**Review by 0xABAB — APPROVED** [review]

File: pkgs/applications/graphics/mypaint/default.nix [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
+  inherit (python2Packages) python pycairo pygobject3 numpy;
 in stdenv.mkDerivation rec {
   name = "mypaint-${version}";
-  version = "1.1.0";
+  version = "1.2.1";
 
-  src = fetchurl {
-    url = "http://download.gna.org/mypaint/${name}.tar.bz2";
-    sha256 = "0f7848hr65h909c0jkcx616flc0r4qh53g3kd1cgs2nr1pjmf3bq";
+  src = fetchgit {
```
Comment (FRidh) [inline pkgs/applications/graphics/mypaint/default.nix]: fetchFromGitHub

Comment (jtojnar) [inline pkgs/applications/graphics/mypaint/default.nix]: `fetchFromGitHub` does not seem to recognize the submodule for me.

Comment (FRidh) [inline pkgs/applications/graphics/mypaint/default.nix]: pass `fetchSubmodules = false;`

Comment (jtojnar) [inline pkgs/applications/graphics/mypaint/default.nix]: Weird, I tried that previously and it did not work, now it seems to be fine.

**Review by pSub — APPROVED** [review]

**Pull request merged**